src/mcts/params.cc

```diff
@@ -49,6 +49,9 @@ const OptionId SearchParams::kMaxPrefetchBatchId{
     "When the engine cannot gather a large enough batch for immediate use, try "
     "to prefetch up to X positions which are likely to be useful soon, and put "
     "them into cache."};
+const OptionId SearchParams::kLogitQEnabledId{
+    "logit-q-enabled", "LogitQEnabled",
```

Comment:
No need for word "enabled" in boolean flags, "logit-q" = on/off is pretty clear too..

Reply:
Or "use-logit-q" is fine too.